client/components/relatedProducts/RelatedProducts.jsx

```diff
@@ -63,10 +64,24 @@ const RelatedProducts = ({ currentProduct, handleCardClick }) => {
     $('.related-products__card').animate(direction, duration);
   };
 
+  const handleOnWheel = (e) => {
+    const $cards = $('.related-products__card');
+    const numOfCards = $cards.length;
+    const outerWidth = $('.related-products')[0].getBoundingClientRect().width;
```

Comment:
nice implementation, maybe give more descriptive variable names to outerWidth and innerWidth, what are they referring to?